docs/contributor/04-30-ci-cd.md

```diff
@@ -1,40 +1,44 @@
+| [`post-istio-operator-build`](https://github.com/kyma-project/test-infra/blob/578e4bb754b1bf12e1964cb96436bd25b7c21996/prow/jobs/kyma-project/istio/istio-manager.yaml#L117)                       | Builds Kyma Istio Operator's image and pushes it to the `prod` registry.                                                                          |
+| [`post-istio-operator-experimental-build`](https://github.com/kyma-project/test-infra/blob/ca3de673baede44bb1167fe54cc43ec7909fe69d/prow/jobs/kyma-project/istio/istio-manager.yaml#L231)          | Builds Kyma Istio Operator's experimental image and pushes it to the `prod` registry.                                                             |
+| [`main integration tests / Istio integration test`](https://github.com/kyma-project/istio/blob/65822779cbf899909088820ab009450e579199f8/.github/workflows/main-integration.yaml#L59)               | Executes the main integration test suite for Istio Manager on a k3d cluster.                                                                      |
+| [`main integration tests / Istio integration test AWS specific`](https://github.com/kyma-project/istio/blob/65822779cbf899909088820ab009450e579199f8/.github/workflows/main-integration.yaml#L103) | Executes the integration test suite specific for AWS Hyperscaler.                                                                                 |
+| [`main integration tests / Istio integration test GCP`](https://github.com/kyma-project/istio/blob/65822779cbf899909088820ab009450e579199f8/.github/workflows/main-integration.yaml#L133)          | Executes the integration test suite specific for GCP Hyperscaler.                                                                                 |
+| [`main integration tests / Wait for image build`](https://github.com/kyma-project/istio/blob/65822779cbf899909088820ab009450e579199f8/.github/workflows/main-integration.yaml#L34)                 | Waits for the operator's image build to finish so that image-dependent jobs can be executed.                                                      |
+| [`main integration tests / Istio upgrade integration tests`](https://github.com/kyma-project/istio/blob/65822779cbf899909088820ab009450e579199f8/.github/workflows/main-integration.yaml#L46)      | Executes the upgrade integration test suite of Istio Module and verifies if the existing release can be successfully upgraded to the new version. |
+| [`main integration tests / Slack Notification`](https://github.com/kyma-project/istio/blob/65822779cbf899909088820ab009450e579199f8/.github/workflows/main-integration.yaml#L193)                  | Sends a Slack notification to the team's channel if any pipelines fail on the main branch.                                                        |
 
 ## Pipelines Running on Pull Requests to Release Branches
```

Comment:
What about the PR to release branch jobs?
They should be the same as the pull requests to main branch, right? 
Can we remove this section and update the header of the section `Pipelines Running on Pull Requests to the Main Branch` to `Pipelines Running on Pull Requests to the Main Branch and Release branches`?